easybuild/easyblocks/t/trinity.py

```diff
@@ -25,6 +25,13 @@
 """
 EasyBuild support for building and installing Trinity, implemented as an easyblock
 
+This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+Homepage:     https://staff.flinders.edu.au/research/deep-thought 
+Authors::     @author: Robert Qiao <rob.qiao@flinders.edu.au>
+License::     GPLv2.0
+ 
+Adapted from trinity.py from easyblock v4.1.0 
+Original Authors:
```

Comment:
@robqiao Rather than adding this block of comments, please just add yourself as a co-author in the list of authors below, something like:

```
@author: Robert Qiao (DeepThought HPC Service, Flinders University, Adelaide, Australia)

Reply:
@boegel Sure ✔️